atlassian_jwt_auth/key.py

```diff
@@ -64,32 +66,34 @@ def __init__(self, base_url):
         if not base_url.endswith('/'):
             base_url += '/'
         self.base_url = base_url
-        self._session = None
+        self._session = self._get_session()
```

Comment:
I don't think it is significant and on second thought I think we don't need a major version release. (I was thinking about a use case that a team has where they change the session to cache differently).